docs/versioned-plugins/codecs/avro-v3.2.0.asciidoc

```diff
@@ -62,17 +62,13 @@ output {
 [id="{version}-plugins-{type}s-{plugin}-options"]
 ==== Avro Codec Configuration Options
 
-This plugin supports the following configuration options plus the <<{version}-plugins-{type}s-{plugin}-common-options>> described later.
```

Comment:
The codecs don't have common options. This is a problem that was fixed in later versions of the plugin docs, but didn't make it into the doc source for earlier plugin versions.